### PR TITLE
Thread race problem pictured

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

**Review by Rudiachenko — APPROVED** [review]
good job

File: pom.xml [hunk 1]
```diff
@@ -8,5 +8,68 @@
...
+                    <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
+                    <encoding>UTF-8</encoding>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: do we need this plugin?

File: src/main/java/com/concurrency/Counter.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.concurrency;
+
+public class Counter {
+    public static final int END_POINT = 100;
```
Comment (Sofasmile) [inline src/main/java/com/concurrency/Counter.java]: let's make this field private and create a method that will check if the `value` is less than `END_POINT`

**Review by Sofasmile — COMMENTED** [review]
great

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**